lib/brcobranca/retorno/cnab240/sicoob.rb

```diff
@@ -0,0 +1,94 @@
+# -*- encoding: utf-8 -*-
+
+require 'parseline'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.